lib/core/src/lib/card-view/components/card-view-textitem/card-view-textitem.component.scss

```diff
@@ -1,204 +1,57 @@
-.adf {
-    &-textitem-edit-icon.mat-icon {
-        font-size: var(--theme-subheading-2-font-size);
-        width: 16px;
-        height: 16px;
-        color: var(--adf-theme-foreground-text-color-025);
+.adf-card-view-textitem {
+    .adf-textitem-error {
```

Comment:
should not be text-item instead of textitem?

Reply:
closing based on https://github.com/Alfresco/alfresco-ng2-components/pull/8995#discussion_r1379735582 comment